client/less/main.less

```diff
@@ -300,6 +300,8 @@ ul {
   }
 }
 
+li.avatar, li.avatar > a { padding:0; margin:0 }
+
```

Comment:
A good rule of thumb when contributing to open source is to retain the style of the project. Ideally it  should have a consistent style throughout and your PR's shouldn't change that.